src/construction.cpp

```diff
@@ -421,35 +421,45 @@ void construction_menu()
                             } else {
                                 result_string = ter_str_id( current_con->post_terrain ).obj().name();
                             }
-                            current_line << color_title << result_string.c_str() << "</color>";
+                            current_line << colorize( result_string, color_title );
                             std::vector<std::string> folded_result_string = foldstring( current_line.str(),
                                     available_window_width );
                             current_buffer.insert( current_buffer.end(), folded_result_string.begin(),
                                                    folded_result_string.end() );
 
                             // display description of the result for multi-stages
                             current_line.str( "" );
-                            current_line << _( "Result: " ) << color_data;
+                            current_line << "Result: ";
```

Comment:
Are there any reasons why you removed the translation of "Result: "?

Reply:
Oh, that's what it does! I see. Apologies. I thought it was related to string construction somehow, but now that you mention it `_()` does ring a bell. Oops.